provider/cmd/pulumi-gen-eks/main.go

```diff
@@ -1426,13 +1426,6 @@ func vpcCniProperties(kubeconfig bool) map[string]schema.PropertySpec {
 				"same VPC as your control plane resources) that are independent of your cluster's " +
 				"`resourcesVpcConfig`.\n\nDefaults to false.",
 		},
-		"externalSnat": {
-			TypeSpec: schema.TypeSpec{Type: "boolean"},
-			Description: "Specifies whether an external NAT gateway should be used to provide SNAT " +
-				"of secondary ENI IP addresses. If set to true, the SNAT iptables rule and off-VPC " +
-				"IP rule are not applied, and these rules are removed if they have already been " +
-				"applied.\n\nDefaults to false.",
-		},
```

Comment:
Should we deprecate this instead of removing it in case users are depending on it?

Probably not related to this PR - but the inconsistency that some of these are `cni...` and others aren't is a little odd.  It actually feels like we most likely want to standardize on *not* including the `cni` prefix (since these are all properties of the VpcCni anyway.  In which case, it might make sense to deprecate the other one instead?  (And file an issue to track aligning the remaining too `cni...` property names with the naming scheme?

Reply:
+1 on deprecating instead. If you want to back this change out and open an issue about the duplication instead, we can address that separately and in the process standardize the properties as Luke suggested.

Reply:
I've backed it out, and am filing a new issue for this.